tutorial/source/gplvm.ipynb

```diff
@@ -297,13 +299,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First, we need to define the output $y$. To predict values of all $48$ genes, we need $48$ Gaussian processes. So in Pyro GP, the required shape for $y$ is `num_GPs x num_data = 48 x 437`."
+    "First, we need to define the output tensor $y$. To predict values for all $48$ genes, we need $48$ Gaussian processes. So the required shape for $y$ is `num_GPs x num_data = 48 x 437`."
```

Comment:
what do the `y` observations actually represent?

Reply:
`y` is the high dimensional output, which is what we want to do dimensional reduction on.

Reply:
i'm not talking about math... i mean the thing that's done in the lab